README.md

```diff
@@ -1,5 +1,7 @@
 # Wundertools
 
+**Note!** This project is no longer in active use. For new projects, use <https://github.com/wunderio/drupal-project> instead.
```

Comment:
Change in active use -> in active development (and maybe state that receives only maintenance fixes).
Change new projects, use -> new projects we recommend.

Reply:
Ok, will push soon, didn't notice this earlier. Thanks!

Reply:
Nice. Could also be an idea to explain in which cases one would want to use this for new projects, to help for example new wunderers making the choice..